bigquery/google/cloud/bigquery/job.py

```diff
@@ -1247,6 +1306,24 @@ def _scrub_local_properties(self, cleaned):
         configuration = cleaned['configuration']['query']
 
         self.query = configuration['query']
+
+    def _copy_configuration_properties(self, configuration):
+        """Helper:  assign subclass configuration properties in cleaned."""
+        self.allow_large_results = _bool_or_none(
+            configuration.get('allowLargeResults'))
+        self.flatten_results = _bool_or_none(
+            configuration.get('flattenResults'))
+        self.use_query_cache = _bool_or_none(
+            configuration.get('useQueryCache'))
+        self.use_legacy_sql = _bool_or_none(
+            configuration.get('useLegacySql'))
+
+        self.create_disposition = configuration.get('createDisposition')
+        self.priority = configuration.get('priority')
+        self.write_disposition = configuration.get('writeDisposition')
+        self.maximum_billing_tier = configuration.get('maximumBillingTier')
+        self.maximum_bytes_billed = configuration.get('maximumBytesBilled')
```

Comment:
Don't you need `_int_or_none` for maximumBytesBilled too?

From: https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest
```
  "maximumBytesBilled": {
     "type": "string",
     "description": "[Optional] Limits the bytes billed for this job. Queries that will have bytes billed beyond this limit will fail (without incurring a charge). If unspecified, this will be set to your project default.",
     "format": "int64"
    },
```